### ISSUE TITLE
oc login fails: The connection to the server oauth-openshift.apps-crc.testing was refused

### ISSUE THREAD

**Issue by lubuxnl** [code-ready/crc#1650]
### General information

  * OS: Linux
  * Hypervisor: KVM
  * Did you run `crc setup` before starting it (Yes/No)? Yes

## CRC version
```bash
CodeReady Containers version: 1.18.0+bb304aa
OpenShift version: 4.6.1 (embedded in binary)
```
  
## CRC status
```bash
CRC VM:          Running
OpenShift:       Degraded
Disk Usage:      14.06GB of 32.72GB (Inside the CRC VM)
Cache Usage:     15.09GB
Cache Directory: /home/luberto/.crc/cache
```

## CRC config
```bash
None
```

## Host Operating System
```bash
NAME="CentOS Linux"
VERSION="8 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="8"
PLATFORM_ID="platform:el8"
PRETTY_NAME="CentOS Linux 8 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:8"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-8"
CENTOS_MANTISBT_PROJECT_VERSION="8"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="8"
```

### Steps to reproduce

  1. crc delete --clear-cache
  2. crc setup
  3. crc start
  4. oc login -u developer -p developer https://api.crc.testing:6443

### Expected
Successful login

### Actual
Error: The connection to the server oauth-openshift.apps-crc.testing was refused - did you specify the right host or port?

### Logs
[crc-start.log](https://github.com/code-ready/crc/files/5505248/crc-start.log)
[oc-login.log](https://github.com/code-ready/crc/files/5505250/oc-login.log)

**Comment by lubuxnl** [comment]
Earlier today I noticed that the time on my CentOS machine hadn't been updated to 'winter' time, so the clock was an hour ahead of the actual time. After having fixed that, I started crc again and this time there was no warning that the cluster operator authentication was still not ready after 2min and I could successfully login.
I don't know why, but apparently the date/time on your host machine must be accurate. Is it possible to have crc detect this during startup and have it issue a warning if the time is not correct?

**Comment by jasonwcc** [comment]
i encountered this many times too.
but stop > setup/start can be a real long process
every time, after reboot system, we have to doi this.

**Comment by cfergeau** [comment]
This issue is 1 year old, a lot have changed since then. Opening a new issue and providing the information the issue template asks for would be useful if you are having problems with crc. If you are having issues related to system reboots, you might want to give a try to 1.34.0 which had some fixes related to this.

**Comment by imperialguy** [comment]
I have the same issue.
